packages/create-contentful-app/template.json

```diff
@@ -0,0 +1,31 @@
+{
+  "package": {
+    "dependencies": {
+      "@contentful/field-editor-single-line": "^0.8.4",
+      "@contentful/field-editor-test-utils": "^0.6.4",
+      "@contentful/forma-36-fcss": "^0.0.35",
+      "@contentful/forma-36-react-components": "^3.39.7",
+      "@contentful/forma-36-tokens": "^0.5.3",
+      "@testing-library/jest-dom": "^5.10.0",
+      "@testing-library/react": "^10.2.1",
+      "@testing-library/user-event": "^12.0.1",
+      "@types/jest": "^25.2.3",
+      "@types/node": "^12.0.0",
+      "@types/react": "^16.9.0",
+      "@types/react-dom": "^16.9.0",
+      "contentful-cli": "1.4.11",
+      "contentful-ui-extensions-sdk": "3.15.0",
+      "inquirer": "^7.2.0",
+      "node-fetch": "^2.6.0",
+      "typescript": "^3.8.0"
+    },
+    "scripts": {
+      "deploy": "echo implement this!!",
+      "configure": "contentful login && npm run create-app-definition",
+      "create-app-definition": "node ./bin/prompt-app-definition",
+      "login": "contentful login",
+      "logout": "contentful logout",
+      "help": "contentful-extension-scripts help"
```

Comment:
We should have our own help

Reply:
Actually, we were planning on using these as shared scripts btn app & extensions